frontend/src/pages/PipelineDetailsTest.test.tsx

```diff
@@ -31,6 +31,12 @@ import * as StaticGraphParser from '../lib/StaticGraphParser';
 import { PageProps } from './Page';
 import PipelineDetails from './PipelineDetails';
 import * as WorkflowUtils from 'src/lib/v2/WorkflowUtils';
+import jsyaml from 'js-yaml';
+import fs from 'fs';
+import YAML from 'yaml';
+
+const v2PipelineSpec_path = 'src/data/test/lightweight_python_functions_v2_pipeline_rev.yaml';
```

Comment:
Use the following name pattern for constants: `V2_PIPELINESPEC_PATH`

Reply:
OK!